client/mysql.cc

```diff
@@ -5031,14 +5032,9 @@ void tee_putc(int c, FILE *file)
 #endif
 #endif
 
-static ulong start_timer(void)
+static ulonglong start_timer(void)
```

Comment:
All callers should change their data types from ulong to ulonglong as well. Fortunately there's just one:
```
static int
com_go(String *buffer,char *line __attribute__((unused)))
{
  ...
  timer=start_timer();
  ...
}
```